src/operations/txbuild.js

```diff
@@ -283,9 +448,17 @@ function makePreorder(fullyQualifiedName: string,
 function makeUpdate(fullyQualifiedName: string,
                     ownerKeyHex: string,
                     paymentKeyHex: string,
-                    zonefile: string) {
+                    zonefile: string,
+                    valueHash: string = '') {
   const network = config.network
-  const valueHash = hash160(Buffer.from(zonefile)).toString('hex')
+  if (valueHash.length === 0) {
+    if (!zonefile) {
+      throw new Error('Need zonefile or valueHash arguments')
```

Comment:
Better to check `valueHash` like: `if (!valueHash) `, since that will detect `null` as well. Then, error throws should be: 
```javascript
return Promise.reject(new Error(..
```

Reply:
Fixed both